mmv1/products/metastore/api.yaml

```diff
@@ -0,0 +1,235 @@
+        description: |
+          Time the AccessPolicy was updated in UTC.
+        output: true
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: 'User-defined labels for the metastore service.'
+      # This is an x-product resource reference.
+      - !ruby/object:Api::Type::String
+        name: 'network'
+        input: true
+        description: |
+          The relative resource name of the VPC network on which the instance can be accessed. It is specified in the following form:
+
+          "projects/{projectNumber}/global/networks/{network_id}".
+      - !ruby/object:Api::Type::String
+        name: 'endpointUri'
+        output: true
+        description: |
+          The URI of the endpoint used to access the metastore service.
+      - !ruby/object:Api::Type::Integer
+        name: 'port'
+        description: |
+          The TCP port at which the metastore service is reached. Default: 9083.
+        default_value: 9083
+      - !ruby/object:Api::Type::String
+        name: 'state'
+        output: true
+        description: |
+          The current state of the metastore service.
+      - !ruby/object:Api::Type::String
+        name: 'stateMessage'
+        output: true
+        description: |
+          Additional information about the current state of the metastore service, if available.
+      - !ruby/object:Api::Type::String
+        name: 'artifactGcsUri'
+        output: true
+        description: |
+          A Cloud Storage URI (starting with gs://) that specifies where artifacts related to the metastore service are stored.
+      - !ruby/object:Api::Type::Enum
+        name: 'tier'
+        description: |
+          The tier of the service.
+        default_value: :TIER_UNSPECIFIED
+        values:
+          - :TIER_UNSPECIFIED
+          - :DEVELOPER
+          - :ENTERPRISE
+      - !ruby/object:Api::Type::NestedObject
+        name: 'maintenanceWindow'
+        description: |
+          The one hour maintenance window of the metastore service.
+          This specifies when the service can be restarted for maintenance purposes in UTC time.
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: 'dayOfWeek'
+            description: |
+              The hour of day (0-23) when the window starts.
+          - !ruby/object:Api::Type::Enum
+              name: 'optionalComponents'
+              description: |
+                The day of week, when the window starts.
+              default_value: :DAY_OF_WEEK_UNSPECIFIED
+              values:
+              - :DAY_OF_WEEK_UNSPECIFIED
+              - :MONDAY
+              - :TUESDAY
+              - :WEDNESDAY
+              - :THURSDAY
+              - :FRIDAY
+              - :SATURDAY
+              - :SUNDAY
+      - !ruby/object:Api::Type::String
+        name: 'uid'
+        output: true
+        description: |
+          The globally unique resource identifier of the metastore service.
```

Comment:
Is this useful for a user to have access to? If it is required to set up another resource to access the service then sure, but I would guess it is a value that the user doesn't care about

Reply:
not that I know of? Probably safe to remove for now